src/asset-swapper/utils/protocol_fee_utils.ts

```diff
@@ -66,28 +67,26 @@ export class ProtocolFeeUtils {
         this._initializeHeartBeat();
     }
 
-    private async _getGasPriceFromGasStationOrThrowAsync(): Promise<BigNumber> {
+    private async _getGasPriceFromGasStationOrThrowAsync(): Promise<void> {
```

Comment:
it looks like a more proper name would be something like `updateGasPriceFromOracleOrThrow`